### ISSUE TITLE
i CAN NOT SELECT MATERIAL:

### ISSUE THREAD

**Issue by arnold-b** [furti/FreeCAD-ArchTextures#24]
Hello, 

if i try to select a material its empty. how can i get it ?

thanks

**Comment by furti** [comment]
Can you tell me the FreeCAD Version you are using?

And Maybe you can give me a short summary of the steps you performed to reproduce the problem.

**Comment by johnrambo85** [comment]
Hi, 
I`ve got the same problem.

- FreeCAD Version 0.17
- OS Lubuntu 16.04

I use the steps you recommend in your Readme.
But I can not choose any material, it is just empty.

Thanks

**Comment by furti** [comment]
Thanks for the Version Information.

The Workbench does only fully work with FreeCAD 0.18 and Python 3. So I would recommend that you update to the newest Python 3 Build available and give it a try.

If you tried to follow the tutorial, I published a fix in the provided "House.FCStd" file right  now. I realized that there was a texture config already inside the file. I think i saved it once when testing something.
